### PR TITLE
add SCIP to solver table

### PR DESCRIPTION
**Pull request by leethargo**

### CODE REVIEW

File: _includes/solver-tables.html [hunk 1]
```diff
@@ -305,10 +305,18 @@
...
-
-
-
+    <tr>
+      <th>
+        <a href="http://scip.zib.de/">SCIP</a>
+        <a href="https://github.com/SCIP-Interfaces/SCIP.jl">(.jl)</a>
+      </th>
+      <td>&#10004;</td> <!-- Linear/Quadratic -->
+      <td></td> <!-- Convex conic -->
```
Comment (mlubin) [inline _includes/solver-tables.html]: Doesn't SCIP do MISOCP? That gets a check mark here.

Comment (leethargo) [inline _includes/solver-tables.html]: That's true, but we don't implement MPB's `AbstractConicModel` and I thought that this is meant by the table column.

Comment (mlubin) [inline _includes/solver-tables.html]: Add a line like [this](https://github.com/JuliaOpt/CPLEX.jl/blob/a07d7099b92753a876a12ddede2265b679550cbe/src/CplexSolverInterface.jl#L33) and you're all set.

Comment (mlubin) [inline _includes/solver-tables.html]: (plus ``supportedcones``)

**Comment by leethargo** [comment]
I added the `ConicModel` now: [#39](https://github.com/SCIP-Interfaces/SCIP.jl/pull/39).

There's a new SCIP release coming up real soon, so we should probably wait for that before doing a new release.

**Comment by chkwon** [comment]
Is this ready to go?

**Comment by leethargo** [comment]
I think so, yes.

**Pull request merged**